packages/pieces/mysql/src/lib/actions/delete-rows.ts

```diff
@@ -0,0 +1,34 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { mysqlCommon, mysqlConnect } from "../common";
+
+export default createAction({
+    name: 'delete_rows',
+    displayName: 'Delete Rows',
+    description: 'Deletes one or more rows from a table',
+    props: {
+        authentication: mysqlCommon.authentication,
+        timezone: mysqlCommon.timezone,
+        table: mysqlCommon.table(),
+        condition: Property.ShortText({
```

Comment:
The same suggestion regarding `Find Row` / `Update Row` can be used here, replacing it with a simple condition for a no-code user. 

Cover delete by single column easily and for more sophisticated case, execute query always an option.